ocaml-lsp-server/src/fmt.ml

```diff
@@ -6,32 +6,6 @@ type command_result =
   ; status : Unix.process_status
   }
 
-module Lazy_fiber : sig
-  type 'a t
-
-  val create : (unit -> 'a Fiber.t) -> 'a t
-
-  val force : 'a t -> 'a Fiber.t
-end = struct
-  type 'a t =
-    { value : 'a Fiber.Ivar.t
-    ; mutable f : (unit -> 'a Fiber.t) option
-    }
-
-  let create f = { f = Some f; value = Fiber.Ivar.create () }
-
-  let force t =
-    let open Fiber.O in
-    let* v = Fiber.Ivar.peek t.value in
-    match v with
-    | Some s -> Fiber.return s
-    | None ->
-      let* v = (Option.value_exn t.f) () in
-      let+ () = Fiber.Ivar.fill t.value v in
-      t.f <- None;
-      v
-end
-
```

Comment:
Was that change intentional? Looks like it produces a build error (I've the same locally): https://github.com/ocaml/ocaml-lsp/pull/398/checks